common/src/main/java/io/netty/util/NetUtil.java

```diff
@@ -264,12 +268,28 @@ public Integer run() {
                             logger.debug("{}: {}", file, somaxconn);
                         }
                     } else {
-                        if (logger.isDebugEnabled()) {
-                            logger.debug("{}: {} (non-existent)", file, somaxconn);
+                        // Try to get from sysctl
+                        Integer tmp = null;
+                        String sysctlCmd = SystemPropertyUtil.get("io.netty.net.sysctl", "sysctl");
```

Comment:
@normanmaurer - FYI. This is to get the max con on systems that don't have `/proc` by default (e.g. MacOS). I can set this property to `null` by default if we don't want to start-up new processes during initialization ... not sure if this will be too heavy weight.